Wrappers/Python/cil/optimisation/algorithms/FISTA.py

```diff
@@ -248,33 +291,41 @@ class FISTA(ISTA):
 
     """
 
-    def set_step_size(self, step_size):
+    def _calculate_default_step_size(self, step_size):
 
-        """Set the default step size
+        """Calculate the default step size if a step size rule or step size is not provided 
```

Comment:
I would return just the 1/Lipschitz constant as default and let the user deal with the constant step size they want to pass outside of this method.